File: leetcode/typescript/[1480]Running-Sum-of-1d-Array.ts
function runningSum(nums: number[]): number[] {
 const result:number[] = [];
 for(let i = 0; nums.length > i ; i++) {
     i <= 0 ? result.push(nums[i]) : result.push(nums[i] + result[i-1])
 }
 return result;
};

File: leetcode/python/[1480]Running-Sum-of-1d-Array.py
from ast import List


class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        result = []
        for i in range(len(nums)):
            curr = 0 if i == 0 else result[i-1]
            result.append(nums[i] + curr)
        return result